multi_map_it_test.go

```diff
@@ -450,3 +450,120 @@ func TestMultiMap_TryLockWithLeaseAndTimeout(t *testing.T) {
 		assert.False(t, b)
 	})
 }
+
+func TestMultiMap_ContainsKey(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		keyExists, err := m.ContainsKey(ctx, "key")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.False(t, keyExists)
+		// assert key value pair
+		it.MustValue(m.Put(ctx, "key", "value"))
+		keyExists, err = m.ContainsKey(ctx, "key")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.True(t, keyExists)
+	})
+}
+
+func TestMultiMap_ContainsValue(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		valueExists, err := m.ContainsValue(ctx, "value")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.False(t, valueExists)
+		// assert key value pair
+		it.MustValue(m.Put(ctx, "key", "value"))
+		valueExists, err = m.ContainsValue(ctx, "value")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.True(t, valueExists)
+	})
+}
+
+func TestMultiMap_MultiMapEntryListener(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		key := "k1"
+		cases := []struct {
+			listenerName string
+			event        hz.EntryEventType
+			setConf      func(*hz.MultiMapEntryListenerConfig)
+			triggerEvent func()
+		}{
+			{
+				listenerName: "EntryAdded",
+				event:        hz.EntryAdded,
+				setConf: func(conf *hz.MultiMapEntryListenerConfig) {
+					conf.NotifyEntryAdded(true)
+				},
+				triggerEvent: func() {
+					ok, err := m.Put(ctx, key, "testValue")
+					if err != nil {
+						t.Fatal(err)
+					}
+					assert.True(t, ok)
+				},
+			},
+			{
+				listenerName: "EntryRemoved",
+				event:        hz.EntryRemoved,
+				setConf: func(conf *hz.MultiMapEntryListenerConfig) {
+					conf.NotifyEntryRemoved(true)
+				},
+				triggerEvent: func() {
+					it.MustBool(m.Put(ctx, key, "testValue"))
+					val, err := m.Remove(ctx, key)
+					if err != nil {
+						t.Fatal(err)
+					}
+					assert.Equal(t, []interface{}{"testValue"}, val)
+				},
+			},
+			{
+				listenerName: "EntryAllCleared",
+				event:        hz.EntryAllCleared,
+				setConf: func(conf *hz.MultiMapEntryListenerConfig) {
+					conf.NotifyEntryAllCleared(true)
+				},
+				triggerEvent: func() {
+					it.MustBool(m.Put(ctx, key, "testValue"))
+					err := m.Clear(ctx)
+					if err != nil {
+						t.Fatal(err)
+					}
+				},
+			},
+		}
+		opSucceed := false
```

Comment:
how about `success` instead of `opSucceed`?

Reply:
Thanks, fixed it at c1204cb